frontend/public/components/cluster-overview.jsx

```diff
@@ -25,126 +25,128 @@ const fetchHealth = () => coFetch(`${k8sBasePath}/healthz`)
   })
   .catch(errorStatus);
 
-const fetchTectonicHealth = () => coFetchJSON('health')
+const fetchConsoleHealth = () => coFetchJSON('health')
   .then(() => ({short: 'UP', long: 'All good', status: 'OK'}))
   .catch(() => ({short: 'ERROR', long: 'The console service cannot be reached', status: 'ERROR'}));
 
 
 const DashboardLink = ({to, id}) => <Link id={id} className="co-external-link" target="_blank" to={to}>View Grafana Dashboard</Link>;
+const getConsoleTitle = isOpenShift => isOpenShift ? 'OpenShift Console' : 'Tectonic Console';
 
-
-const Graphs = ({namespace}) => <React.Fragment>
-  <div className="group">
-    <div className="group__title">
-      <h2 className="h3">Health</h2>
-      <DashboardLink id="qa_dashboard_k8s_health" to="/grafana/dashboard/db/kubernetes-cluster-health?orgId=1" />
-    </div>
-    <div className="container-fluid group__body">
-      <div className="row">
-        <div className="col-md-3 col-sm-6">
-          <Status title="Kubernetes API" fetch={fetchHealth} />
-        </div>
-        <div className="col-md-3 col-sm-6">
-          <Status title="Tectonic Console" fetch={fetchTectonicHealth} />
-        </div>
-        <div className="col-md-3 col-sm-6">
-          <Status
-            title="Alerts Firing"
-            name="Alerts"
-            query={`sum(ALERTS{alertstate="firing", alertname!="DeadMansSwitch" ${namespace ? `, namespace="${namespace}"` : ''}})`}
-            href="/alertmanager/#/alerts" target="_blank" rel="noopener"
-          />
-        </div>
-        <div className="col-md-3 col-sm-6">
-          <Status
-            title="Crashlooping Pods"
-            name="Pods"
-            query={`count(increase(kube_pod_container_status_restarts${namespace ? `{namespace="${namespace}"}` : ''}[1h]) > 5 )`}
-            href={`/k8s/${namespace ? `ns/${namespace}` : 'all-namespaces'}/pods?rowFilter-pod-status=CrashLoopBackOff`}
-          />
-        </div>
-      </div>
-    </div>
-  </div>
-
-  { !namespace &&
+const Graphs = requirePrometheus(({isOpenShift, namespace}) => {
+  const consoleTitle = getConsoleTitle(isOpenShift);
+  return <React.Fragment>
     <div className="group">
       <div className="group__title">
-        <h2 className="h3">Control Plane Status</h2>
-        <DashboardLink to="/grafana/dashboard/db/kubernetes-control-plane-status?orgId=1" />
+        <h2 className="h3">Health</h2>
+        {!isOpenShift && <DashboardLink id="qa_dashboard_k8s_health" to="/grafana/dashboard/db/kubernetes-cluster-health?orgId=1" />}
```

Comment:
Since the `to` path is used on multiple places, with different fragments, wouldn't it be better to construct it with a function ?

Reply:
I'm a little hesitant to change this since I don't have a good way to test.

I kind of prefer that they're opaque URLs, though. It's not clear they'll always follow the same pattern.

Reply:
yeah I understand your concern. On the other hand if they change the pattern it would be more easy if the path is stored in constant.
But that could be done as a followUp :)